WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
I prefer the current setup but support using "The Hun" as a nickname. As it is however there is no justification for making the change you did, given that Attilid works and is consistent with similar other cases, while "the Hun" is the definition of a nickname.